src/templates/page.js

```diff
@@ -18,26 +19,36 @@ import {
   MainWrapper,
   LeftColumn,
   MiddleColumn,
-  Content,
-  PageHeader
+  Content
 } from "../components/core/components";
 
-import * as DesignSystem from "@livechat/design-system";
+import { Header as PageHeader } from "../components/core/Page";
+import Rating from "../components/core/Rating";
+
 import { Headings, CodeBlocks } from "../components/extensions";
 import SEO from "../components/core/seo";
 import RichMessagePreview from "../vendors/rich-message-preview.min.js";
-import useLocalStorage from "../hooks/useLocalStorage";
-import { VersionProvider } from "../contexts/version";
+
+import { useLocalStorage, useRating } from "../hooks";
+import { VersionProvider, RatingProvider } from "../contexts";
 import { SCROLL_OFFSET } from "../constant";
 
 const components = {
-  ...DesignSystem,
   ...CodeBlocks,
   ...Headings,
   RichMessagePreview,
   Link
 };
 
+const StyledRating = styled(Rating)`
+  width: 270px;
+  margin: 50px 0 0 calc(50% - 135px);
```

Comment:
You can just wrap rating component with some div position flex, justify-content: center and you have your rating centered without margin calculation